tasks/snyk-code-test.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: snyk-code-test
+spec:
+  description: >-
+    Static code security test with snyk
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  params:
+    - name: shared-secret
```

Comment:
nit: I'd suggest using the upper case and underscore separated format for the params, e.g. `SHARED_SECRET` in this case.